Dockerfile

```diff
@@ -81,7 +82,11 @@ COPY 10-cleanup-htcondor.sh /etc/osg/image-cleanup.d/
 COPY 10-htcondor.conf /etc/supervisord.d/
 COPY 50-main.config /etc/condor/config.d/
 RUN chmod 755 /bin/entrypoint.sh
- 
+
+RUN [[ -n $TIMESTAMP ]] || TIMESTAMP=$(date +%Y%m%d-%H%M); \
```

Comment:
Not sure how `docker build` handles `ARG` expansion; did you try out this build with an empty `TIMESTAMP`?

Reply:
Yeah, this works but I'm revisiting this since we should separate manual builds out from our automated builds.